commands_base/__init__.py

```diff
@@ -9,9 +9,13 @@
 import verboselogs
 
 
-def main(project_name: str):
+verboselogs.install()
```

Comment:
Oh!  No, this isn't what I was asking for.

I was asking for this to be moved into `__main__.py`, and for project_name to be passed in as an argument from `__main__.py` as well.  In other words, to leave the original signature of main() intact and to leave things where they were.

Reply:
so __main__.py was not being used, but I've broken it up so that it's a separate function that's used as the cli entrypoint